adsdocmatch/refstatus.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+import requests
+from adsputils import load_config
+
+proj_home = os.path.realpath(os.path.join(os.path.dirname(__file__), "../"))
+config = load_config(proj_home=proj_home)
+
+class FailedQueryException(Exception):
+    pass
+
+
+class QueryConfigException(Exception):
+    pass
+
+
+class NoResultsException(Exception):
+    pass
+
+
+class RefStatusException(Exception):
+    pass
+
+
+def refereed_status(bibstem):
```

Comment:
Sorry Matt. I probably did not realize that you had name it this. Since you are going to implement more logic to infere if a bibstem is matchable or not, I would rename so that it would display this is not only refereed, but also other logic.

Reply:
I'll change this to "matchable_status".